plugin-repos/march-madness/manager.py

```
@@ -0,0 +1,907 @@
+            if dt.tzinfo is None:
+                start_time_utc = dt.replace(tzinfo=pytz.UTC)
+            else:
+                start_time_utc = dt.astimezone(pytz.UTC)
+            local = start_time_utc.astimezone(pytz.timezone("US/Eastern"))
+            game_date = local.strftime("%-m/%-d")
+            game_time = local.strftime("%-I:%M%p").replace("AM", "am").replace("PM", "pm")
+        except (ValueError, AttributeError):
+            pass
+
+        # Period / clock for live games
+        situation = comp.get("situation", {})
```

Ruff (0.15.2)

[error] Local variable `situation` is assigned to but never used

Remove assignment to unused variable `situation`

(F841)